### ISSUE TITLE
[Feedback] Kubernetes examples

### ISSUE THREAD

**Issue by billpratt** [dotnet/eShopOnContainers#139]
I noticed a Kubernetes branch on this repo. I'm running production ASP.NET Core APIs in Kubernetes and would like to help contribute. Can you provide a quick update on the status of this branch?

A few of my thoughts/roadmap ideas (more to come):

* Getting started guide w/ using [minikube](https://kubernetes.io/docs/getting-started-guides/minikube/)
* Examples using [https://github.com/kubernetes/helm](helm)

**Comment by CESARDELATORRE** [comment]
Hi Bill!
The Kubernetes branch is in experimental/tests mode and is not up-to-date with the latest code we have in DEV or even at MASTER branch.
We'll focus on orchestrators once we finish with other "core features" like Domain Events, Integration Events, Event Bus, resiliency techniques, etc.
One we get to a more stable version of eShopOnContainers, we'll focus on Orchestrators in Azure:
- Kubernetes (in ACS)
- Docker Swarm (in ACS)
- Mesos DC/OS (in ACS)
- Azure Service Fabric

Thanks for your feedback on Kubernetes, I'll take those ideas and add it as possible features for our eShopOnContainers roadmap.

Btw, you might find interesting the following preliminary (early draft) documentation that we created about Kubernetes:
https://github.com/dotnet/eShopOnContainers/blob/eShopOnKubernetes/KUBERNETES.md  

Btw, I saw the email thread with you and Bill Wagner. Let's connect further through email or the ISSUES while you review eShopOnContsainers, ok? - We'd love to get your feedback. :)  

Thanks,
Cesar.
